ci/npm-grunt.yml

```diff
@@ -13,16 +13,17 @@ jobs:
     strategy:
       matrix:
         node-version: [12.x, 14.x, 16.x]
-    
+
     steps:
     - uses: actions/checkout@v2
 
     - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
+      uses: actions/setup-node@v2
       with:
         node-version: ${{ matrix.node-version }}
+        cache: npm
+
+    - run: npm ci
```

Comment:
`npm ci` will fail if repository does have package-lock.json file. So, I think we should not change the `npm install` to `npm ci` as it would be breaking for many repos

Reply:
True, but a bunch of the others are using `npm ci` already. If the projects have manually excluded the package-lock.json file from their repo, then they also know to change this